### PR TITLE
Feat/add GitHub workflow to publish package

### PR DESCRIPTION
**Pull request by yuhaoju**
Add Github workflow to publish package.

### CODE REVIEW

**Review by mr-pinzhang — APPROVED** [review]

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+jobs:
+  publish-gpr:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 10
+          registry-url: https://npm.pkg.github.com/
+          scope: '@adrise'
```
Comment (mr-pinzhang) [inline .github/workflows/release.yml:16]: will this `scope` have effects? it seems mismatched with what scope it contains in package.json

**Comment by mr-pinzhang** [comment]
@yuhaoju also, please reset your branch HEAD to `tubi/master` branch.

**Review by mr-pinzhang — CHANGES_REQUESTED** [review]
I'll approve later.

**Comment by yuhaoju** [comment]
@elsefalse I've talked with Heng and moved Hls.js to this repo https://github.com/adRise/web_ott_npm_private_packages/tree/master/packages/hls.js, forgot to close this PR, sorry.

**Pull request closed without merging**